.changeset/light-badgers-sparkle.md

```diff
@@ -0,0 +1,7 @@
+---
+"@justeattakeaway/pie-textarea": minor
+"pie-storybook": minor
+---
+
+[Added] - `assistiveText` property, `status` property with the possible values `default`, `success` and `error`
+[Added] - storybook controls for the `assistiveText` and `status` properties
```

Comment:
Is it possible to split these into separate files? I _think_ it results in nicer formatting